components/brave_sync/brave_sync_service_impl.cc

```diff
@@ -365,12 +366,20 @@ void BraveSyncServiceImpl::OnSaveInitData(const Uint8Array& seed,
 
   sync_words_.clear();
   DCHECK(!seed_str.empty());
-  if (prev_seed_str == seed_str) { // reconnecitng to previous sync chain
+
+  if (prev_seed_str == seed_str) { // reconnecting to previous sync chain
     sync_prefs_->SetPrevSeed(std::string());
   } else if (!prev_seed_str.empty()) { // connect/create to new sync chain
     bookmark_change_processor_->Reset(true);
     sync_prefs_->SetPrevSeed(std::string());
-  } // else {} no previous sync chain
+  } else {
+    // This is not required, because when there is no previous seed, bookkmarks
+    // should not have a metadata. However, this is done by intention, to be
+    // a ramedy for cases when sync had been reset and prev_seed_str had been
```

Comment:
s/ramedy/remedy/

Reply:
thanks, force-pushed fix for grammatical errors